src/DeviceStatusPage/DeviceLiveEventsCard.tsx

```diff
@@ -76,6 +76,7 @@ function watchDeviceEvents({
           on: 'device_connected',
           device_id: deviceId,
         },
+        policy: `connectiontrigger-${deviceId}`,
```

Comment:
I'm not sure why in this file you added a reference to a `connectiontrigger-${deviceId}` policy: I don't see any code that creates such policy.
I guess you can remove these additions from the PR.